app/src/test/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityLocalTest.kt

```diff
@@ -166,6 +171,32 @@ class QuestionPlayerActivityLocalTest {
     }
   }
 
+  @Test
+  @Config(qualifiers = "port")
+  fun testQuestionPlayer_portrait_submitCorrectAnswer_correctIsAnnounced() {
```

Comment:
For both this & the exploration variant it might be a good idea to add a new test each for verifying that the announcement does *not* happen if there is feedback since that's an important requirement being implemented.

Reply:
Done, added more tests.